src/attributes.rs

```diff
@@ -17,11 +17,70 @@ use crate::types::AnyTypeEnum;
 // REVIEW: Should Attributes have a 'ctx lifetime?
 /// Functions, function parameters, and return types can have `Attribute`s to indicate
 /// how they should be treated by optimizations and code generation.
-#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+#[derive(Clone, Copy)]
 pub struct Attribute {
     pub(crate) attribute: LLVMAttributeRef,
 }
 
+impl std::fmt::Debug for Attribute {
```

Comment:
I used the Debug and PartialEq instance during debugging. I can remove them again, but then when something fails you only see an address, which is not helpful at all.